interceptors/validator/validator.go

```diff
@@ -41,9 +63,13 @@ func validate(req interface{}) error {
 // UnaryServerInterceptor returns a new unary server interceptor that validates incoming messages.
 //
 // Invalid messages will be rejected with `InvalidArgument` before reaching any userspace handlers.
-func UnaryServerInterceptor() grpc.UnaryServerInterceptor {
+func UnaryServerInterceptor(validateAll ...bool) grpc.UnaryServerInterceptor {
```

Comment:
Do we need to make this variadic now? Seems we could just make it a single bool and force the user to supply it right? Could you please add what the meaning of the variable is to the comment above the method? Thanks!

Reply:
Agree! Since we are working on v2, keeping the signature compatible with v1 is not actually necessary. Use a clearer single argument instead. Also add comments for the new argument.

Please review again. Thanks!